packages/mdl-theme/README.md

```diff
@@ -0,0 +1,106 @@
+# mdl-theme
```

Comment:
README looks great so far! 

I think we should add docs on theming using SASS (using the functions/mixins, and overriding the default vars) as well as using custom properties.